thats_the_way.py

```diff
@@ -0,0 +1,17 @@
+from os import listdir
+from os.path import isfile, join
+
+
+def that_the_way(path):
+    file_names = [f for f in listdir(path) if isfile(join(path,f)) and 'deep' in f]
```

Comment:
just return the list comprehension, you don't have to assign it to a different variable